### ISSUE TITLE
Error in mount_disks role

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/ansible-hydra#267]
When I run a playbook I have previously used, I'm getting the following error from the `mount_disks` role:
```
TASK [mount_disks : format volume(s)] ********************************************************************************************
failed: [52.4.91.185] (item={'key': u'/dev/xvdf', 'value': {u'fs': u'ext4', u'group': u'ubuntu', u'mount': u'/opt', u'perm': 755, u'owner': u'ubuntu', u'delete': True, u'type': u'gp2', u'size': 40}}) => {"failed": true, "item": {"key": "/dev/xvdf", "value": {"delete": true, "fs": "ext4", "group": "ubuntu", "mount": "/opt", "owner": "ubuntu", "perm": 755, "size": 40, "type": "gp2"}}, "msg": "Device /dev/xvdf not found."}
	to retry, use: --limit @/Users/mark/Documents/workspace/_no_backup_/deploying_hydra/tufts-cm/tufts-ubuntu.retry
```

Specifically I'm getting the error `Device /dev/xvdf not found`

I successfully ran the same playbook without errors on August 7, 2017.  It appears that the tasks that mount the `/dev/xvdf` volume were removed from `machine.yml` in [#245] (https://github.com/curationexperts/ansible-hydra/pull/245/files) on August 11, 2017.

I need help understanding what I'm supposed to do now to get an `/opt` volume?

**Comment by acozine** [comment]
Yes, this was a mistake that should be rectified by the current `multi_sec_groups` branch. You can work around it by associating a disk manually - I hope to have a PR ready and merged today.

**Comment by acozine** [comment]
Functionality fixed by #271. Device_name is part of the data dictionary for `ec2_instances` - see `roles/launch_ec2/defaults/main.yml` for an example.
